system/jlib/jtrace.hpp

```diff
@@ -383,6 +385,23 @@ constexpr std::initializer_list<TraceOption> roxieTraceOptions
     TRACEOPT(traceAcknowledge),
 };
 
+constexpr std::initializer_list<TraceOption> thorTraceOptions
+{ 
+    TRACEOPT(traceNone),
+    TRACEOPT(traceStandard),
+    TRACEOPT(traceDetailed),
+    TRACEOPT(traceMax),
+    TRACEOPT(traceHttp),
+    TRACEOPT(traceSockets),
+    TRACEOPT(traceCassandra),
+    TRACEOPT(traceMongoDB),
+    TRACEOPT(traceCouchbase),
+    TRACEOPT(traceFilters),
+    TRACEOPT(traceKafka),
+    TRACEOPT(traceJava),
```

Comment:
looks like a lot of these should be removed/are not going to be used by Thor (cassandra/mongodb/counchbase ..)